lib/deploy/repository.rb

```diff
@@ -21,75 +12,53 @@ def prepare!(tag)
   private
 
   def tag_exists?
-    repo.tags.map(&:name).include? @tag
+    require 'open3'
```

Comment:
How many times do we call this method? I wonder if it could be nicer to put requires at the top of the file